synfig-core/src/synfig/synfig_export.h

```diff
@@ -27,7 +27,7 @@
 
 #ifdef _MSC_VER
 // We need this export only for MSVC. Even on MinGW, it breaks linkning.
-#ifdef synfig_EXPORTS
+#ifdef libsynfig_EXPORTS

```

Comment:
Is this correct?

Reply:
Yes. MSVC automatically adds the `%library_name%_EXPORTS` definition when creating a shared library.